docs/agent-api.asciidoc

```diff
@@ -783,3 +783,37 @@ apm.clearPatches('timers')
 
 apm.clearPatches(['timers'])
 ----
+
+[[apm-current-trace-ids]]
+==== `apm.currentTraceIds`
+
+[small]#Added in: #2.17.0
+
+Produces an object containing `trace.id` and either `transaction.id` or `span.id` when a current transaction or span is available.
+When no transaction or span is available it will return an empty object.
+This enables log correlation to APM traces with structured loggers.
+
+[source,js]
+----
+{
+  "trace.id": "abc123",
+  "transaction.id": "abc123"
+}
+// or ...
+{
+  "trace.id": "abc123",
+  "span.id": "abc123"
+}
+----
+
+All current trace id objects,
+including the empty form,
+include a `toString()` implementation.
+This enables log correlation to APM traces with text-only loggers.
+
+[source,js]
+----
+"trace.id=abc123,transaction.id=abc123"
```

Comment:
this and a few more instances in docs below need updating now

Reply:
Oh, good catch. Should be fixed now. 👍